OrbitLinuxTracing/CMakeLists.txt

```diff
@@ -67,21 +67,45 @@ target_link_libraries(OrbitLinuxTracing PUBLIC
         CONAN_PKG::abseil
         CONAN_PKG::libunwindstack)
 
+set(CMAKE_SKIP_BUILD_RPATH FALSE)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+set(CMAKE_INSTALL_RPATH $ORIGIN)
+
+
+add_executable(OrbitLinuxTracingTestPuppet)
```

Comment:
Two things about this. 

First, I don't like the name "puppet", it should more clearly state what it's doing, i.e., dlopen.

Second, the only place where we have this kind of targets to profile for manual tests is `OrbitTest`. I would keep it like that and move this to `OrbitTest`.

Reply:
Alright.